### PR TITLE
Bookstore: Styling

### PR DESCRIPTION
**Pull request by Timbar09**
**In this PR, I added styling to the website**

### CODE REVIEW

File: src/styles/Navbar.module.css [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  gap: 2.5rem;
+}
+
+.logo {
+  font-family: 'Monoton', cursive;
+  font-size: 2em;
+  line-height: 1.4;
+  color: var(--clr-blue);
+  padding-bottom: 0.25em;
+}
```
Comment (binhussen) [inline src/styles/Navbar.module.css:19]: - Great Job For having this beautiful UI 👍👍 however you have to modify it to the given [zepline](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) design as it is part of the [requirement](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/styling.md) 👌
### here is your design
Screenshot 2023-03-03 at 12 12 07 PM

### here is [zepline](https://app.zeplin.io/project/5b35a9e13227086040f8eb75/screen/5b695e29bb8c844f118f9378) design
Screenshot 2023-03-03 at 12 02 18 PM

**Review by binhussen — CHANGES_REQUESTED** [review]
Hi @Timbar09 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- All linter checks passed ✔️
- Used Git flow ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/styles/Navbar.module.css (hunk 1 above)
Comment (Timbar09) [reply src/styles/Navbar.module.css:19]: Hi, @binhussen. Thanks for the feedback.

The design has features like the completion progress, chapters, Update progress functionality, comments, and edit. 

On this project, we do not have these features. We only have the book title, author, and the delete button. I cannot implement those other features when the project does not include them.

Please let me know how you think I should go forward.

**Review by DeliceLydia — APPROVED** [review]
## Hi @Timbar09, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Highlights
- Addressed all changes requested


### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Timbar09** [comment]
Thanks, @DeliceLydia.

**Pull request merged**